### DISCUSSION TITLE
如何在触发节点的dblclick事件时，不触发2次click？

### DISCUSSION THREAD

**Discussion by AmieJing** [antvis/G6#3432] [Q&A]
如何在触发节点的dblclick事件时，不触发2次click？
![image](https://user-images.githubusercontent.com/37952540/148728711-d7151a84-3b41-45fc-8c85-7da6014492be.png)

**Comment by Yanyan-Wang** [comment]
这个是 MDN 标准哈，DOM 的 dblclick 也会触发两次 click。可以通过在 on click 中自己加一个定时器，来决定是否执行 click 的逻辑
